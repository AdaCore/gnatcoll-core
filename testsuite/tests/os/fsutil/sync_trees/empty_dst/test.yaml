description: Test for GNATCOLL.OS.FSutil.Sync_Trees. Scenario with an empty
             destination directory
driver: build_run_assert_and_diff
data:
  - "src_top_dir"
  - "dst_top_dir"
source_dirs:
  - "../"
control:
    - [SKIP, "env.is_cross", "Copy of tests directories fail from windows host to board (eng/toolchain/gnatcoll-core#35)"]