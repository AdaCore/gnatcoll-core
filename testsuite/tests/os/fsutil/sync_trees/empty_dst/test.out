Test for GNATCOLL.OS.FSUtil.Sync_Trees
OK   test.adb:15
Source tree
===========
DIR src_top_dir/dir-0
DIR src_top_dir/dir-0/dir-0
DIR src_top_dir/dir-0/dir-1
DIR src_top_dir/dir-0/dir-2
DIR src_top_dir/dir-1
DIR src_top_dir/dir-1/dir-0
DIR src_top_dir/dir-1/dir-1
DIR src_top_dir/dir-1/dir-2
DIR src_top_dir/dir-2
DIR src_top_dir/dir-2/dir-0
DIR src_top_dir/dir-2/dir-1
DIR src_top_dir/dir-2/dir-2
FILE src_top_dir/file-0
FILE src_top_dir/file-1
FILE src_top_dir/file-2

Destination tree
================
DIR dst_top_dir/dir-0
DIR dst_top_dir/dir-0/dir-0
DIR dst_top_dir/dir-0/dir-1
DIR dst_top_dir/dir-0/dir-2
DIR dst_top_dir/dir-1
DIR dst_top_dir/dir-1/dir-0
DIR dst_top_dir/dir-1/dir-1
DIR dst_top_dir/dir-1/dir-2
DIR dst_top_dir/dir-2
DIR dst_top_dir/dir-2/dir-0
DIR dst_top_dir/dir-2/dir-1
DIR dst_top_dir/dir-2/dir-2
FILE dst_top_dir/file-0
FILE dst_top_dir/file-1
FILE dst_top_dir/file-2

Content checking
================
<=== TEST PASSED ===>
